src/main/scala/ch/epfl/bluebrain/nexus/kg/resolve/ProjectResolution.scala

```diff
@@ -34,11 +35,15 @@ abstract class ProjectResolution[F[_]: Monad](cache: DistributedCache[F], static
     new Resolution[F] {
 
       private val resolution = cache.resolvers(ref).map { res =>
-        val result = res.map {
-          case _: StaticResolver       => staticResolution
-          case r: InProjectResolver    => InProjectResolution[F](r.ref, resources)
-          case r: InAccountResolver    => InAccountResolution[F](r.accountRef, resources, cache)
-          case _: CrossProjectResolver => CrossProjectResolution[F](resources, res)
+        val result = res.filterNot(_.deprecated).toList.sortBy(_.priority).map {
+          case _: StaticResolver    => staticResolution
+          case r: InProjectResolver => InProjectResolution[F](r.ref, resources)
+          case r: InAccountResolver =>
+            val projects = cache.projects(r.accountRef).map(_.map(_ -> r.resourceTypes).toList)
```

Comment:
`toList.distinct` ? Since there is room for duplication and we should avoid calling twice the resolution for the same project and resource type